pages/common/clido.md

```diff
@@ -0,0 +1,36 @@
+# clido
+
+> Save-state TODO app for the terminal.
+> For more info, visit the Clido Wiki: <https://codeberg.org/Oglo12/clido/wiki>.
```

Comment:
The documentation link should always be prefixed with "More information:".
https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#layout

```suggestion
> More information: <https://codeberg.org/Oglo12/clido/wiki>.
```